scripts/ci/gitlab/nextest.toml

```diff
@@ -120,6 +120,10 @@ store-failure-output = true
 # To check if you added your test to a filter and group correctly, use this command:
 # cargo nextest show-config test-groups --config-file scripts/ci/gitlab/nextest.toml --features runtime-benchmarks --workspace --locked --release --manifest-path ./bin/node/cli/Cargo.toml
 # For the non-default [[profile.default.overrides]] i.e. [[profile.local.overrides]] add `--profile local` to the above command
+filter = 'test(/(^ui$|_ui|ui_)/)'
```

Comment:
That won't work. `[[profile.default.overrides]]` means that last one will override the prev one. If you need to add another group you have to create another profile. Or mb there is another way. Check docs for nextest.

Reply:
It does. My bad.